src/patternfly/components/Form/styles.scss

```diff
@@ -0,0 +1,462 @@
+  --pf-c-form__check__label--PaddingLeft: 0;
+  --pf-c-form__check--MarginRight--addons: var(--pf-global--spacer--sm);
+
+  // Label
+  --pf-c-form__label--Color: var(--pf-global--Color--dark-200);
+  --pf-c-form__label--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-form__label--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-form__label--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__label--MarginRight: var(--pf-global--spacer--sm);
+
+  // Label - required
+  --pf-c-form__label--Left--required: -12px;
+  --pf-c-form__label--Top--required: 2px;
+
+  // Label - disabled
+  --pf-c-form__label--Color--disabled: var(--pf-global--Color--disabled);
+
+  // Label Group
+  --pf-c-form__label-group__item--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Form check
+  --pf-c-form__check__item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-form__check--Padding: var(--pf-global--spacer--sm);
+
+  // Addons
+  --pf-c-form__check__addons--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Checkbox 
+  --pf-c-form__checkbox--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-form__checkbox--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-form__checkbox--PaddingRight: 0;
+  --pf-c-form__checkbox--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-form__checkbox--PaddingLeft: 0;
+
+  // Radio
+  // == Change me ==
+  --pf-c-form__radio--MarginRight: var(--pf-global--spacer--md);
+
+  // Fieldset
+  --pf-c-form__fieldset--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Select
+  --pf-c-select__arrow--BorderWidth: var(--pf-global--BorderWidth--md);
+
+  // Change these to globals
+  // States
+  --pf-c-form--Color--error: var(--pf-global--danger-color--dark);
+
+  // Helpers
+  --pf-c-form__helper-text--MarginTop: var(--pf-global--spacer--sm);
+  --pf-c-form__helper-text--FontSize: var(--pf-global--FontSize--sm);
+
+  // Textarea
+  --pf-c-form__textarea--Padding: var(--pf-global--spacer--sm);
+
+  // Actions
+  --pf-c-form__form-actions--MarginRight: var(--pf-global--spacer--md);
+
+  // Tooltip
+  --pf-c-tooltip--MarginLeft: var(--pf-global--spacer--xs);
+
+  // Inline
+  --pf-c-form--MarginRight--inline: var(--pf-global--spacer--md);
+}
+
+// Base form
+.pf-c-form {
+
+  // Form group - pf-c-form__group
+  &__group {
+
+    // Inline layout
+    &.pf-m-inline {
+      display: flex;
+      flex-flow: row wrap;
+      > * {
+        flex: 0;
+        margin-right: var(--pf-c-form--MarginRight--inline);
+      }
+
+      // Label outside of pf-c-form__check
+      > .pf-c-form__label {
+        flex: 1 0 100%;
+        margin: 0;
+      }
+
+      .pf-c-form__label {
+        white-space: nowrap;
+      }
+    }
+  }
+
+  // Form label - pf-c-form__label
+  &__label {
+    display: flex;
+    align-items: baseline;
+    // using padding here to prevent flicker
+    padding-bottom: var(--pf-c-form__label--PaddingBottom);
+    margin: 0;
```

Comment:
I am ok with the preference of a padding over a margin, but do we need a margin zero here?